projects/plugins/jetpack/extensions/plugins/blaze/icon.js

```diff
@@ -0,0 +1,39 @@
+import { SVG, Path, Rect, G } from '@wordpress/components';
+
+export default function BlazeIcon() {
+	return (
+		<SVG width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/SVG">
```

Comment:
Can we use a smaller SVG height to avoid this extra space?

<img width="542" alt="image" src="https://user-images.githubusercontent.com/1044309/208980668-5112be3b-1ddd-48be-8423-9c2bd10e2d1e.png">